model/clusters_mgmt/v1/optional_cluster_capability_type.model

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright (c) 2025 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Optional cluster capabilities that can be disabled.
+enum OptionalClusterCapability {
```

Comment:
There is an open discussion whether use or not enums. Let's wait till we have an agreement.

Reply:
@ahitacat where is that discussion happening?

Reply:
@davegord I saw it as an open question in this [service/uhc-clusters-service!9360](https://gitlab.cee.redhat.com/service/uhc-clusters-service/-/merge_requests/9360/diffs#note_14773785)

Reply:
We've decided to go with plain strings.